mdx-models/src/main/java/com/mx/path/model/mdx/accessor/profile/SecurityQuestionBaseAccessor.java

```diff
@@ -0,0 +1,46 @@
+package com.mx.path.model.mdx.accessor.profile;
+
+import com.mx.path.core.common.accessor.API;
+import com.mx.path.core.common.accessor.AccessorMethodNotImplementedException;
+import com.mx.path.core.common.gateway.GatewayAPI;
+import com.mx.path.core.common.gateway.GatewayClass;
+import com.mx.path.gateway.accessor.Accessor;
+import com.mx.path.gateway.accessor.AccessorConfiguration;
+import com.mx.path.gateway.accessor.AccessorResponse;
+import com.mx.path.model.mdx.model.profile.SecurityQuestions;
+
+/**
+ * Accessor base for profile security questions
+ */
+@GatewayClass
+@API(specificationUrl = "https://developer.mx.com/drafts/mdx/profile/index.html#security-questions")
+public class SecurityQuestionBaseAccessor extends Accessor {
+  public SecurityQuestionBaseAccessor() {
+  }
+
+  @Deprecated
```

Comment:
I think you want this on the class definition?

Reply:
Wait... why is this being added and marked as deprecated?

Reply:
Must have been a copy past error. fixed it.